### ISSUE TITLE
How to delete ghost admin?

### ISSUE THREAD

**Issue by AdalbertTheCreator** [Ballsdex-Team/BallsDex-DiscordBot#318]
I created my bot using instructions from wiki, but when I tried to register me as admin, it acted like the bot already had an admin. When I tried to use command from https://github.com/Ballsdex-Team/BallsDex-DiscordBot/issues/273#issuecomment-2150146303, powershell only reports syntax error. Here is the screenshot of it: 
![Zrzut ekranu 2024-07-23 183609](https://github.com/user-attachments/assets/b93d42e6-6355-4fd7-9a0a-3cc71a6decf5)

**Comment by imtherealF1** [comment]
@AdalbertTheCreator hi, can you try opening a cmd prompt and typing in this:
docker compose exec postgres-db psql -U ballsdex ballsdex -c "TRUNCATE TABLE \"user\";"
after you type that, it should only return TRUNCATE TABLE, meaning that the table got deleted, and then you will need to go to your localhosts/admin/init, and make a new account.

**Comment by AdalbertTheCreator** [comment]
> @AdalbertTheCreator hi, can you try opening a cmd prompt and typing in this: docker compose exec postgres-db psql -U ballsdex ballsdex -c "TRUNCATE TABLE "user";" after you type that, it should only return TRUNCATE TABLE, meaning that the table got deleted, and then you will need to go to your localhosts/admin/init, and make a new account.

It only shows this
![obraz_2024-09-03_172011495](https://github.com/user-attachments/assets/3aa2218f-19f2-49ae-b89e-4648511c58a2)

**Comment by AdalbertTheCreator** [comment]
> @AdalbertTheCreator hi, can you try opening a cmd prompt and typing in this: docker compose exec postgres-db psql -U ballsdex ballsdex -c "TRUNCATE TABLE "user";" after you type that, it should only return TRUNCATE TABLE, meaning that the table got deleted, and then you will need to go to your localhosts/admin/init, and make a new account.

This happens
![obraz_2024-09-04_173913413](https://github.com/user-attachments/assets/8f34612a-2d6c-43aa-865d-b2d641cd5452)

**Comment by AdalbertTheCreator** [comment]
> You can see on your screenshot that the quotation marks ends before the end of the prompt, that's why you need two different quotation marks. `user` must be enclosed in double quotation marks, so the whole query must be enclosed with single quotations.
> 
> TLDR: ` docker compose exec postgres-db psql -U ballsdex ballsdex -c 'TRUNCATE TABLE "user";'`
> 
> You can also open `psql` without the `-c` and write the query in the prompt.

This doesn't work either
![Zrzut ekranu 2024-09-29 091938](https://github.com/user-attachments/assets/f823395f-f7e8-49ff-9c27-31b41d09055f)

**Comment by laggron42** [comment]
That's odd, maybe another powershell fuckery?

Try only `docker compose exec postgres-db psql -U ballsdex ballsdex`, then paste `TRUNCATE TABLE "user";` and type enter, maybe the prompt will actually work...

**Comment by AdalbertTheCreator** [comment]
> That's odd, maybe another powershell fuckery?
> 
> Try only `docker compose exec postgres-db psql -U ballsdex ballsdex`, then paste `TRUNCATE TABLE "user";` and type enter, maybe the prompt will actually work...

Ok, that's better, but it's still not that
![Zrzut ekranu 2024-10-07 153158](https://github.com/user-attachments/assets/ad6c6957-074a-4731-b1de-c3f09806329a)

**Comment by laggron42** [comment]
No that worked, that's the expected output. You can now go back to `/admin/init` to create the first admin

**Comment by AdalbertTheCreator** [comment]
yeah
